### ISSUE TITLE
Bug: Pyblish / Collected with errors

### ISSUE THREAD

**Issue by Alice51000** [quadproduction/OpenPype#519]
### Before posting a new ticket, have you looked through the documentation to find an answer?

Yes I have

### Have you looked through the existing tickets to find any related issues ?

Not yet

### Author of the bug

Alice Sammut

### Version

3.14.7

### What platform you are running OpenPype on?

Windows

### Current Behavior:

Hello,

I tried to export a shot tvpaint as PSD and publish but I can't, it has a collected with errors message. I never had that type of problem before, we already had published this one. We have a delivery today, if it can be solved as soon as you can it would be perfect.

I don't know if it's linked but we had several resolution issue and had to change it. I think this shot might be concerned but I don't remember for sure !

It's the Wizard With a Gun project > shot 90 > task colo

![Pyblisherror](https://github.com/quadproduction/OpenPype/assets/134387402/a19595d7-2330-446d-81f7-e42e3cb9248f)
![Capture_erreur_publish](https://github.com/quadproduction/OpenPype/assets/134387402/0292a90a-e6a4-410a-9daa-d876d1c63a51)

### Expected Behavior:

1. Create RenderLayer
2. Pyblish
3. uncheck in and out mark
4. uncheck validate scene settings

### What type of bug is it ?

Happened only once in a particular configuration

### Which project / workfile / asset / ...

Wizard With a Gun III / ANIMATION / SHOT90

### Steps To Reproduce:

1. launch tvpaint
2. Create RenderLayer
3. Pyblish
4. Meet the bug 🕷

**Comment by ccaillot** [comment]
I've tried to reproduce the error several times on my side and it seems that's the error does not appears on the 3.15, can you retry and tell me if you are able to reproduce the error on your own ?

**Comment by hfarre** [comment]
I can confirm
The error doesn't seem to appear

We can close this ticket and keep an eye on future error of this type if they appear in 3.15
